### ISSUE TITLE
parameters setting for fine-tuning the model on kitti dataset using pre-trained model on cityscapes?

### ISSUE THREAD

**Issue by jasonTuZq** [mrharicot/monodepth#27]
Hi,
i want to ask what's your parameters setting for fine-tuning the model on kitti dataset using pre-trained model on cityscapes? Do you just use the default parameters in your code for the results in your paper?
is the following command the right way to do so?
python monodepth_main.py --mode train --model_name xxx --data_path dataset/kitti/ --filenames_file utils/filenames/kitti_train_files.txt --log_directory tmp --checkpoint_path pre-trainedModel/model_cityscapes/model_cityscapes --retrain

Thanks for your help!

**Comment by mrharicot** [comment]
Hi,

Yes, the "fine-tuning" was more of a re-training on a different dataset using the same parameters.
The code has by default the parameters from the paper.

**Comment by jasonTuZq** [comment]
![image](https://user-images.githubusercontent.com/21214673/28453884-a7081590-6e2b-11e7-962c-8b7183e1fc57.png)
Hi, i fine tuned the model using the default parameters and the loss oscillated very much as shown in the attached png, is this something like you got?

**Comment by mrharicot** [comment]
Yes, this looks normal!
You can probably kill the learning rate more aggressively and get less oscillations.

**Comment by MeghaMaheshwari** [comment]
Hey, I too got quite a lot of oscillations on the training set even though I trained on the kitti dataset using the default parameters. Is this expected, as the results of evaluation still match the ones from your paper

**Comment by mrharicot** [comment]
Yes, it is to be expected, the loss is very noisy.
If you use the smoothing in Tensorboard it should help to visualize the convergence.
